### PR TITLE
Console buttons

### PR DESCRIPTION
**Pull request by gsn9**
I made the console buttons work, but not the `Draw WKT String` button yet as I wanted to bring up potentially removing the button altogether. Do we really want/need this button? I know Trent said he liked that we kept it in but I would argue since we removed the line and polygon drawing tools we should remove this as well since it's not going to be linked to any footprints or data. What are your guy's thoughts on this? 

Solves issue #48

### CODE REVIEW

**Comment by amystamile** [comment]
I say remove the wkt stuff altogether to avoid confusion.

**Review by amystamile — COMMENTED** [review]
Sorry I reviewed this a couple a days ago and forgot to click submit. Just one comment.

**Comment by amystamile-usgs** [comment]
@gsn9, What is the current status of this PR?

File: app/src/js/AstroMap.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import AstroProj from "./AstroProj";
 import LayerCollection from "./LayerCollection";
 import { getItemCollection, setNumberMatched, setMaxNumberPages, getCurrentPage, setCurrentPage, setFeatures} from "./ApiJsonCollection";
 import { MY_JSON_MAPS } from "./layers";
-import stacLayer from 'stac-layer/src/index.js';
+// import stacLayer from 'stac-layer/src/index.js';
```
[the remark this answers is not in the document]
Comment (amystamile-usgs) [reply app/src/js/AstroMap.js:7]: Still confused why this is commented out.

[the remark this answers is not in the document]
Comment (gsn9) [reply app/src/js/AstroMap.js:7]: Huh, weird my comments aren't posting, not sure what's up with that. It's commented out as I can't compile the app with it in the code. Not sure if it's something with how maybe my paths are set up or something else. Let's see if the screenshot will show this time. 
image

**Review by amystamile — APPROVED** [review]

**Pull request merged**